docs/pages/distribution/app-size.mdx

```diff
@@ -0,0 +1,108 @@
+
+There are various types of build artifacts that serve different purposes, and they are almost all larger than what users will see when they download your app from a store. This is because these builds are not optimized to target specific devices like they would when downloading from a store, but rather they typically include all of the code and resources that your app needs to run on a wide range of devices.
+
+## iOS apps
+
+The download size on the App Store of a minimal React Native app is just under 4 MB.
+
+There are two types of iOS build artifacts that you will interact with: APPs and IPAs.
+
+### `.app` (iOS application bundle)
+
+This is the actual application bundle for your app. When you download and install a build of your app into an iOS Simulator, you are downloading the `.app` bundle. These can either target specific architectures or be universal binaries. The size of your `.app` doesn't necessarily tell you too much about what the download size of your app will be on the store. You can't install a `.app` file directly to a physical iOS device.
+
+### `.ipa` (iOS App Store Package)
+
+IPA files are [ZIP](https://en.wikipedia.org/wiki/ZIP)) files that include the `.app` bundle and other resources that are needed to run the app on an iOS device, and they are used for various types of distribution including App Store, Ad Hoc, Enterprise, and TestFlight. They include security and code signing information, such as the provisioning profile and entitlements. The App Store will take care of processing the IPA file and splitting it into smaller binaries for each device type, and so the size of the IPA also does not represent the download size of your app.
+
+### Determining iOS app download and install size
+
+Typically, what app developers care about the "download size" on the App Store (what the users see in the store listing when they go to download the app). This will be the size of the split IPA that is generated by the store from your universal IPA.
+
+The only truly accurate way to see what your final app size will be shipped to users is to upload your app to the stores and download on a physical device. You can get accurate estimates from TestFlight: on [App Store Connect](https://appstoreconnect.apple.com/), navigate to TestFlight and select your build by clicking on the build number, then switch to the **Build Metadata** tab and click **App File Sizes**. You will see a list of estimated download and install sizes depending on the device type. Actually install sizes may also vary slightly depending on the iOS version on the device.
+
+## Android apps
+
+{/* todo: add info about minimum size of Android app on store */}
```

Comment:
@aleqsio - did you end up looking into this at all, or just on the ios side? if not, could you? 🙏